app.py

```diff
@@ -14,12 +14,7 @@
 app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
 db = SQLAlchemy(app)
 
-from services import (  # noqa: E402
-    update_or_create_category,
-    update_or_create_subcategory,
-    update_or_create_criterion,
-    update_or_create_link,
-)
+import services  # noqa: E402
```

Comment:
I needed to do this so I could patch the outgoing API call to the app script in our tests – let me know if there's a better way!

Reply:
This is what I do too 🤷‍♀️

Reply:
This makes sense to me. I'm not sure how to get around this.